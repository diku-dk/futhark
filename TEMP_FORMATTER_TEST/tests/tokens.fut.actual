cb2a6e6f1da89262ae1abb5b1a4a8337
