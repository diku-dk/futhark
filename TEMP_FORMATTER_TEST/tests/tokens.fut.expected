2fe2d4e0ce3272cf8a4484b86b8f47bd
